Source/msg.cpp

```diff
@@ -3013,11 +3001,9 @@ uint32_t ParseCmd(int pnum, const TCmd *pCmd)
 	case CMD_TSPELLXY:
 		return OnTargetSpellTile(pCmd, player);
 	case CMD_OPOBJXY:
-		return OnOperateObjectTile(pCmd, player);
 	case CMD_DISARMXY:
-		return OnDisarm(pCmd, player);
 	case CMD_OPOBJT:
-		return OnOperateObjectTelekinesis(pCmd, player);
+		return OnObjectTileAction(*reinterpret_cast<const TCmdLoc *>(pCmd), player);

```

Comment:
This might be the opposite of what I was thinking, so in case it was bad communication on my part here is a code sample instead :D

```cpp
	case CMD_OPOBJXY:
		return OnObjectTileAction(pCmd, player, ACTION_OPERATE);
	case CMD_DISARMXY:
		return OnObjectTileAction(pCmd, player, ACTION_DISARM);
	case CMD_OPOBJT:
		return OnObjectTileAction(pCmd, player, ACTION_OPERATETK, false);
```